packages/google_maps_flutter/google_maps_flutter_platform_interface/pubspec.yaml

```
@@ -3,7 +3,7 @@ description: A common platform interface for the google_maps_flutter plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.4
+version: 2.0.5
```

A minor version should be updated since there's a publish API change, so
```suggestion
version: 2.1.0
```